### ISSUE TITLE
Tabs component doesn't work with keyboard only navigation on Safari

### ISSUE THREAD

**Issue by xavortm** [10up/component-library#100]
**Describe the bug**
On Safari, I am unable to navigate between tabs if I don't use my mouse. I am able to tab inside the content area only, but not to the tab elements. If I click on a horizontal tab, I am able to move left and right with the keyboard, but not on a vertical tab. I recorded a GIF showing this together with the key presses:

**To Reproduce**
1. Open safari
2. Go to the demo page
3. Try to go to the last tab with just your keyboard

**Expected behavior**
To work same as Chrome and Firefox

**Screenshots**
![Screenshot 2022-02-24 at 20 01 21](https://user-images.githubusercontent.com/2004664/155581537-28403b97-baa8-4e10-be36-a64f45ad20e2.gif)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Safari 
 - Version 15.1 (17612.2.9.1.20)

**Additional context**
Please list more relevant information here.

**Comment by xavortm** [comment]
False alarm, this one is not a bug. I didn't knew Safari is not supporting it by default - https://articulate.com/support/article/How-to-Enable-Tab-Key-Navigation-on-a-Mac

After enabling A11y tab highlight, I am able to navigate with keyboard alone.
